github/actions/client.go

```diff
@@ -259,7 +269,7 @@ func (c *Client) NewActionsServiceRequest(ctx context.Context, method, path stri
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.ActionsServiceAdminToken))
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", c.ActionsServiceAdminToken))
```

Comment:
why do we need to change this?

Reply:
We don't, sorry 😅 that's a leftover from a bug that left scratching our heads for a while.